packages/table/test/columnHeaderCellTests.tsx

```diff
@@ -143,22 +143,18 @@ describe("<ColumnHeaderCell>", () => {
     });
 
     describe("Reorder handle", () => {
-        const REORDER_HANDLE_CLASS = "reorder-handle";
+        const REORDER_HANDLE_CLASS = "bp-table-reorder-handle-target";
```

Comment:
prefer not using `bp-table-` prefix for test-only class. be obvious that this is not a real `Table` class, just a hook used for these tests. i was wondering if it belongs in `Classes`, then I found its usage in `mount()` below.